src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -106,7 +106,8 @@
 
       <_NET70RuntimePackVersion>7.0.$(VersionFeature70)</_NET70RuntimePackVersion>
       <_NET70TargetingPackVersion>7.0.$(VersionFeature70)</_NET70TargetingPackVersion>
-      <_NET70WebAssemblyPackVersion>7.0.$(VersionFeature70)</_NET70WebAssemblyPackVersion>
+      <!-- TODO: Once .NET 8.0.X has released, update these version numbers to 8.0.$(VersionFeature80) -->
+      <_NET70WebAssemblyPackVersion>8.0.0-preview.7.23375.9</_NET70WebAssemblyPackVersion>
```

Comment:
I see a net 8 version on line 101 and it's used on line 638.

Reply:
Dug into the original change to confirm that the 8.0 pack was used for 6 and 7.  Question, should they all use the 9.0 packages or should 6/7/8 use 8.0 and only net9.0 use the latest package?  For now, we're going with the former and can live with that but it'd be good to confirm what's expected long term.

Reply:
Thank you for the update! I didn't realized yesterday that the net7 property is not needed...

Long term plan is that .NET 6/7/8 should use 8.0 package, .NET 9 should use 9.0, etc.

Reply:
Given that there is no pratical difference I'd lean towards using the 9.0 versions unless/until they diverge but it shouldn't matter right now.